introduction.md

```diff
@@ -1 +1,6 @@
-Hello master kobra kai. My name is Patrick Ronayne. I like Javascript.
\ No newline at end of file
+Hello master kobra kai. My name is Patrick Ronayne. I like Javascript.
+*javascript
+*ruby
+*java
+*c++
+*SQL
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?